pages/linux/turbostat.md

```diff
@@ -0,0 +1,24 @@
+# turbostat
+
+> Report processor frequency and idle statistics.
+> More information: <https://www.linux.org/docs/man8/turbostat.html>.
```

Comment:
```suggestion
> More information: <https://manned.org/turbostat.8>.
```
When there isn't an official documentation, we prefer manned.org over other websites for manual pages, according to our style guide.